app_server/views/service.pug

```diff
@@ -1,43 +1,215 @@
+                    tr.active
+                      td Doris H
+                      td
+                        a(href='') 0491 570 156
+                      td 2017/12/21
+                    tr
+                      td Bobby N
+                      td
+                        a(href='') 0491 570 156
+                      td 2017/12/15
+                    tr.active
+                      td Steven C
+                      td
+                        a(href='') 0491 570 156
+                      td 2017/12/4
+                    tr
+                      td Patrick H
+                      td
+                        a(href='') 0491 570 156
+                      td 2017/11/22
+                    tr.active
+                      td Edward T
+                      td
+                        a(href='') 0491 570 156
+                      td 2017/11/8
+                    tr
+                      td Lisa S
+                      td
+                        a(href='') 0491 570 156
+                      td 2017/11/3
+                    tr.active
+                      td Rose E
+                      td
+                        a(href='') 0491 570 156
+                      td 1650 BC
         .row
-            .col-md-8
-                h2
-                    | Do you have a bed available?
-    form(action="/addService", method="post", role="form")
-        .form-group
-            .row
-                .col-md-1.col-xs-3.text-center
-                    label(for="hasBeds")
-                        | Yes
-                    .spacer-sm
-                        input(type="checkbox", value="available", id="hasBeds", name="available")
-                        
-                .col-md-1.col-xs-3(align="center")
-                    label(for="maleBeds")
-                        | Male
-                    input.form-control(style="width: 3.5em;", type="text", id="maleBeds")
-                .col-md-1.col-xs-3(align="center")
-                    label(for="femaleBeds")
-                        | Female
-                    input.form-control(style="width: 3.5em;", type="text", id="femaleBeds")
-                .col-md-1.col-xs-3(align="center")
-                    label(for="otherBeds")
-                        | Unspecified
-                    input.form-control(style="width: 3.5em;", type="text", id="otherBeds")
-                .col-md-1(align="center")
-                    label(for="totalBeds")
-                        | Total
-                    #totalBeds.form-control-static 0
-                .col-md-7.col-xs-12
-                    label(for="description")
-                        | Description
-                    textarea.form-control(type="text", rows=5, id="description", name="description")
-        .form-group
-            .text-right
-                input(type="submit", value="Submit").btn.btn-default
+          .col-xs-12
+            .thumbnail
+              .panel-body(style='height: 500px')
```

Comment:
If you were not aware inline styles is considered bad practice. Personally I don't mind this especially for our project.

Reply:
I didn't know this before, I will change it for the future.